### ISSUE TITLE
Create a 3.0 branch

### ISSUE THREAD

**Issue by blink1073** [jupyterlab/extension-examples#115]
- Create a 3.0 branch
- Update all extensions using `python -m jupyterlab.upgrade_extension` from https://github.com/jupyterlab/jupyterlab/pull/8950
- Update `.github/workflows/main.yaml` to:
  - use `--pre jupyterlab`
  - run `pip install -e .` in each folder instead of ` jlpm run build` and `jupyter labextension install .`

**Comment by blink1073** [comment]
I plan to make a script and do this in one shot when I get back from vacation, next Tuesday.

**Comment by blink1073** [comment]
WIP upgrader script:

```python
import os
import os.path as osp
import shutil
import subprocess
import sys

def get_exts(dname):
    paths = []
    for path in os.listdir(dname):
        if path in ['node_modules', '.git']:
            continue
        full_path = osp.join(dname, path)
        if osp.exists(osp.join(full_path, 'setup.py')):
            continue
        package_path = osp.join(full_path, 'package.json')
        if osp.exists(package_path):
            paths.append(full_path)
        elif osp.isdir(full_path):
            paths.extend(get_exts(full_path))
    return paths

for ext in get_exts('.'):
    print(ext)
    subprocess.run([sys.executable, '-m', 'jupyterlab.upgrade_extension', '--no-input', ext])
    for fname in ['.prettierignore', '.prettierrc', 'LICENSE']:
        os.remove(osp.join(ext, fname))
    for dname in ['.github', '_temp_extension']:
        shutil.rmtree(osp.join(ext, dname))
    break
```

**Comment by blink1073** [comment]
Also working on https://github.com/jupyterlab/jupyterlab/pull/8974 as part of this

**Comment by jtpio** [comment]
Fixed by #119.
